src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -979,8 +983,18 @@ private async Task EntityMiddleware(DispatchMiddlewareContext dispatchContext, F
                     entityContext.State.PutBack(lockHolderMessages);
                 }
 
+                // mitigation for ICM358210295 : un-suspend an entity if the last continue-as-new happened at least 10 seconds ago

```

Comment:
nitpick: can we say this more directly? For example: resume entity processing if it has been suspended for at least 10 seconds

Reply:
Sure, I agree.